src/cpp/src/visual_language/qwen2vl/classes.hpp

```diff
@@ -16,7 +16,7 @@ class VisionEncoderQwen2VL : public VisionEncoder {
 public:
     using VisionEncoder::VisionEncoder;
 
-    EncodedImage encode(const ov::Tensor& image, const ov::AnyMap& config_map) override;
+    EncodedImage encode(ov::InferRequest& encoder, const ov::Tensor& image, const ov::AnyMap& config_map) override;
```

Comment:
This class also has `m_vision_embeddings_merger`, but not sure whether it's a bottleneck